h2o-r/h2o-package/R/connection.R

```diff
@@ -335,7 +334,7 @@ h2o.connect <- function(ip = "localhost", port = 54321, strict_version_check = T
    do.call(h2o.init, c(startH2O=FALSE, config))
  } else {
    # Pass arguments directly
-   h2o.init(ip=ip, port=port, strict_version_check=strict_version_check,
+   h2o.init(ip=ip, port=port, startH2O=FALSE, strict_version_check=strict_version_check,
```

Comment:
it's interesting this reversal between R and Py implementations:
- in Py, `h2o.init()` calls `h2o.connect()`
- in R, `h2o.connect()` calls `h2o.init()`…, hence the issue at the origin of this ticket.

this looks like the right way to fix it in this context.